Resources/MobiledgeXSettings.asset

```diff
@@ -15,7 +15,18 @@ MonoBehaviour:
   sdkVersion:
   orgName: 
   appName:
-  appVers:
+  appVers: 
   authPublicKey: 
-  region:
-  logType:
+  region: Nearest
+  logType: 1
+  edgeEventsConfig:
+    latencyThresholdTriggerMs: 120
+    latencyTestPort: 0
+    latencyConfig:
+      updatePattern: 0
+      updateIntervalSeconds: 30
+      maxNumberOfUpdates: 0
```

Comment:
See above comment ^

My maxNumberOfUpdates in Swift is 0 by default as well. However, 0 means update infinitely

Reply:
Updated the comment to "0 means updates will continue through the app life time"